docs/usage.md

```diff
@@ -58,6 +59,10 @@ The Swagger input processor expects that the property `swagger` is defined in or
 
 The response payload and `body` parameters, since it is a JSON Schema variant, is [interpreted as a such](./interpretation_of_JSON_Schema.md).
 
+## Generate models from MetaModels
```

Comment:
Nitpick: Just to ensure people don't get the wrong idea, `MetaModels` can sound like a custom solution out there. However, if we don't treat it as a "own name", it gives the idea that you can build from something abstract or behind (meta).
Feel free to take it or leave it 👍 
```suggestion
## Generate models from meta models
```

Reply:
I changed the namings, I like it 👍